docker/entrypoint.sh

```diff
@@ -81,4 +89,4 @@ if [ -n "${APP_BASE_SHARE_URL}" ];then
   cat /tmp/app.config.json > ./app.config.json
 fi
 
-nginx -g "daemon off;"
+nginx -g "daemon off;" -c /nginx.conf
```

Comment:
why redirecting the config to the root folder? this will again make the ACA config stand out of all other apps while we are trying to unify everything